ui/app/components/ui/unit-input/index.scss

```diff
@@ -39,7 +39,9 @@
 
     color: #4d4d4d;
     border: none;
-    max-width: 22ch;
+    max-width: 15ch;
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
I like the changed but the ellipsis never shows for me in Firefox; does it for you?

Reply:
Also, oddly enough, I see the `input` element being assigned a width:

<img width="961" alt="8ch" src="https://user-images.githubusercontent.com/46655/98010237-12b76c80-1dbc-11eb-90fa-e305da22c36b.png">

Reply:
> ellipsis never shows for me in Firefox; does it for you?

It does for values with more 15+ character length. Try adding 9+ integers as the input value and I believe is shows up.

The element width is dynamic with the currency suffix. I set `max-width` so the PR description needs to be updated, thanks for pointing that out.